coap-service/coap_service_api.h

```diff
@@ -47,6 +47,8 @@ extern "C" {
 #define COAP_SERVICE_OPTIONS_EPHEMERAL_PORT     0x04
 /** Coap interface selected as socket interface */
 #define COAP_SERVICE_OPTIONS_SELECT_SOCKET_IF   0x08
+/** Register to COAP multicast groups */
```

Comment:
On yet another hand do we implement much of RFC 7390 yet? Do we have anything in the "Request Acceptance and Response Suppression Rules" for multicast? Random response delays?

If we don't do much, maybe we should be more cautious about accepting multicast at all, as it might not behave sensibly for most resources. Maybe that needs a separate "allow multicast" flag on a per-resource basis.